From: Aaron J. Seigo <aseigo@kde.org>
Subject: attempt to fix dual head support
Origin: backport http://websvn.kde.org/?view=rev&revision=1182543
Bug: https://bugs.kde.org/show_bug.cgi?id=156475
Applied-Upstream: 4.5.3

convert the number first to a string, otherwise it doesn't append anything.
noticed by Oliver oin the BR

--- a/plasma/desktop/shell/main.cpp
+++ b/plasma/desktop/shell/main.cpp
@@ -91,7 +91,7 @@ KDE_EXPORT int kdemain(int argc, char **
 
     QByteArray appName = "plasma-desktop";
     if (associatedScreen > 0) {
-        appName.append("-screen-").append(associatedScreen);
+        appName.append("-screen-").append(QByteArray::number(associatedScreen));
     }
 
     KAboutData aboutData(appName, 0, ki18n("Plasma Workspace"),
